### ISSUE TITLE
Failure with sdwebiamge 3.8.2 request images

### ISSUE THREAD

**Issue by duckls** [rs/SDWebImage#1715]
### New Issue Checklist

* [x] I have read and understood the [CONTRIBUTING guide](https://github.com/rs/SDWebImage/blob/master/.github/CONTRIBUTING.md)
* [x] I have read the [Documentation](http://cocoadocs.org/docsets/SDWebImage/)
* [x] I have searched for a similar issue in the [project](https://github.com/rs/SDWebImage/issues) and found none

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           | e.g. ios / tvos
 Platform Version        | e.g. 8.0~10.1
 SDWebImage Version      | e.g. 3.8.2
 Integration Method      | e.g. carthage / cocoapods / manually
 Xcode Version           | e.g. Xcode 8.1
 Repro rate              | e.g. all the time (100%) / sometimes x% / only once
 Repro with our demo prj | e.g. does it happen with our demo project?
 Demo project link       | e.g. link to a demo project that highlights the issue

### Issue Description and Steps

？Ask baidu picture (http://img5.imgtn.bdimg.com/it/u=2253484101, 1253799674 21 & gp & FM = = 0. JPG)Failure, other images can be.

**Comment by duckls** [comment]
http://img5.imgtn.bdimg.com/it/u=2253484101,1253799674&fm=21&gp=0.jpg  完整路径

**Comment by JeepHan** [comment]
I have the same question.

**Comment by bpoplauschi** [comment]
Please reopen when the issue description contains all the required info.
